tellorflex/methods.py

```diff
@@ -145,7 +144,7 @@ def reward_reporter():
         return Seq([
             Assert(is_governance),
             Cond(
-                [Or(
+                [And(
```

Comment:
I'm not sure this should be changed. for example, if `Txn.application_args[1]` == 1, then the assert fails and the transaction reverts

Reply:
I stand corrected